pkg/monitor/dynkeepalived.go

```diff
@@ -166,15 +173,17 @@ func handleBootstrapStopKeepalived(kubeconfigPath string, bootstrapStopKeepalive
 				"consecutiveErr": consecutiveErr,
 			}).Info("handleBootstrapStopKeepalived: detect failure on API")
 		} else {
+			if consecutiveErr > bootstrapApiFailuresThreshold { // Means it was stopped
+				bootstrapStopKeepalived <- started
+			}
 			consecutiveErr = 0
 		}
 		if consecutiveErr > bootstrapApiFailuresThreshold {
 			log.WithFields(logrus.Fields{
 				"consecutiveErr":                consecutiveErr,
 				"bootstrapApiFailuresThreshold": bootstrapApiFailuresThreshold,
 			}).Info("handleBootstrapStopKeepalived: Num of failures exceeds threshold")
-			bootstrapStopKeepalived <- true
-			return
+			bootstrapStopKeepalived <- stopped
```

Comment:
'stop' message will be sent every second after API goes down, right?

Reply:
well, It doesn't matter because at the MCO side if there is no running Keepalived  process it just display a msg to log

Reply:
right